webapp/WEB-INF/conf/plugins/myfiles_context.xml

```diff
@@ -5,6 +5,8 @@
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd" >
 
 
-       <bean id="myfiles.storage" class="fr.paris.lutece.plugins.myfiles.service.storage.impl.MinioFileStorage" />
+    <bean id="myfiles.storage" class="fr.paris.lutece.plugins.myfiles.service.storage.impl.DatabaseFileStorage" />
```

Comment:
As Minio server need to be configured before installing the plugin, it seems better to switch to a local implementation by default. In order to highlight this point, I put this change in its own commit.